go.mod

```diff
@@ -39,6 +39,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/vmware-tanzu/image-registry-operator-api v0.0.0-20230526154708-f67dac7c805f
```

Comment:
Any reason for not moving it to the current require block around L21 (govmomi, ncp, vmop/api are all there, so I think this belongs there)?

Reply:
Nothing in particular, `go mod` must have added it in that place. Refactored.